locale/en_US.json

```
@@ -2772,6 +2772,50 @@
+  "Gestione Collane": "Series Management",
+  "Gestisci le collane e le serie di libri": "Manage book series and collections",
+  "Collane totali": "Total series",
+  "Libri nelle collane": "Books in series",
+  "Nessuna collana trovata. Aggiungi una collana a un libro per iniziare.": "No series found. Add a series to a book to get started.",
+  "Collane": "Series",
+  "Serie e collane di libri": "Book series and collections",
+  "Rinomina collana": "Rename series",
+  "Rinomina": "Rename",
+  "Unisci con altra collana": "Merge with another series",
+  "Nome collana di destinazione": "Target series name",
+  "Unisci": "Merge",
```

Biome (2.4.7)

[error] The key Unisci was already declared.

(lint/suspicious/noDuplicateObjectKeys)